spec/datadog/ci/test_spec.rb

```diff
@@ -32,7 +32,7 @@
 
     context 'when given a block' do
       subject(:trace) { described_class.trace(tracer, span_name, options, &block) }
-      let(:span) { Datadog::Span.new(tracer, span_name) }
+      let(:span) { Datadog::SpanOperation.new(span_name, tracer: tracer) }
```

Comment:
Should we also rename this to `let(:span_operation)` or `let(:operation)`, to match other specs?